ssn/ssn_separated/ssn.ttl

```diff
@@ -531,22 +532,9 @@ ssn:Drift rdf:type owl:Class ;
 
 
 ###  http://www.w3.org/ns/ssn/FeatureOfInterest
-ssn:FeatureOfInterest rdf:type owl:Class ;
-                      rdfs:subClassOf [ rdf:type owl:Restriction ;
-                                        owl:onProperty ssn:hasProperty ;
-                                        owl:someValuesFrom ssn:Property
-                                      ] ,
-                                      [ rdf:type owl:Restriction ;
-                                        owl:onProperty ssn:hasProperty ;
-                                        owl:allValuesFrom ssn:Property
-                                      ] ;
-                      dc:source """skos:exactMatch 'feature' [O&M] 
-		                    http://www.opengeospatial.org/standards/om""" ;
-                      rdfs:comment "A feature is an abstraction of real world phenomena (thing, person, event, etc)." ;
-                      rdfs:isDefinedBy "http://www.w3.org/ns/ssn" ;
-                      rdfs:label "Feature of Interest" ;
-                      rdfs:seeAlso "http://www.w3.org/2005/Incubator/ssn/wiki/SSN_Skeleton#Skeleton" .
-
+#sosa:FeatureOfInterest
+#  rdfs:subClassOf [ owl:onProperty ssn:hasProperty ; owl:someValuesFrom ssn:Property ] ; # TODO: need resolution, is it ssn:hasProperty, or ssn:hasProperty ?
+#  rdfs:subClassOf [ owl:onProperty ssn:hasProperty ; owl:allValuesFrom ssn:Property ]  . # TODO: need resolution, is it ssn:hasProperty, or ssn:hasProperty ?
```

Comment:
need resolution, is it ssn:hasProperty, or sosa:hasProperty ?
This is currently discussed in a separate thred, will just comment these lines for now.
See https://lists.w3.org/Archives/Public/public-sdw-wg/2017Feb/0478.html